src/Accelerators/NNPA/Compiler/NNPACompilerUtils.cpp

```diff
@@ -282,10 +282,8 @@ void addPassesNNPA(mlir::OwningOpRef<mlir::ModuleOp> &module,
         // Optimizations at ZLow after normalizing MemRefs.
         pm.addPass(zlow::createZLowRewritePass());
         // The createZLowStickExpansion pass may create parallel constructs,
-        // they need to be handled here.
-        if (!nnpaDisableCompilerStickUnstick && enableParallel)
-          pm.addPass(mlir::createConvertSCFToOpenMPPass());
-
+        // but we don't need to handle it here as it will be processed when
+        // lowering Krnl to LLVM.
```

Comment:
I think we can remove this comment completely. There is no context to understand it once we removed `createConvertSCFToOpenMPPass ()`

Reply:
Will remove as requested